### ISSUE TITLE
CS2: mirv_viewmodel

### ISSUE THREAD

**Issue by ChetdeJong** [ripieces/advancedfx#941]
Since left hand was added in CS2 it brought new issue.

Apparently looks like viewmodel is now part of demo (not sure though). 
Now it's not possible to change viewmodel anymore when watching demos. 
So every player has different viewmodel and left/right hand preference, which is I would rate as nightmare for video production.

Is it technically possible to hack these settings to force them to be something else?
Basically like `mirv_cvar_hack` was in CSGO.

However I think right/left hand might be tricky/impossible to fix since it plays animation when player changes hand.

Would be much appreciated if you can have a look to see if it's even possible to implement.

**Comment by dtugend** [comment]
The viewmodel itself is simply offset and mirrored on x-z-plane when left-handed (you can tell from text on the side of the glock e.g), however the switchhands animation is recognized in a whole lot of places, so I doubt it would make sense to override the mirroring (notable effort but not impossible of course).
It's not as simple as changing / overriding some cvars from what I saw with a quick check.
Same applies to viewmodel positioning.

**Comment by dtugend** [comment]
Currently not planned, too complicated, would block other tasks I want to work on for HLAE still. Valve itself is probably still working on it, but I don't think they will remove or fix it the way you want either tbh.
